src/styles/_header.scss

```
@@ -37,5 +37,11 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition-duration: 300ms;
+    transition-property: color;
```

```suggestion
    transition: color .3s;
```
Можно сокращать запись